VCFv4.3.tex

```diff
@@ -326,132 +327,147 @@ \subsubsection{Fixed fields}
-  with optional values in the format: $<$key$>$=$<$data$>$[,data]. 
-  INFO keys must match the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed.
-  Arbitrary keys are permitted, although the following sub-fields are reserved (albeit optional):
-\begin{itemize}
-  \item AA : ancestral allele
-  \item AC : allele count in genotypes, for each ALT allele, in the same order as listed
-  \item AD, ADF, ADR: read depths for each allele; total (AD), on the forward (ADF) and the reverse (ADR) strand (Integer, Number=R)
-  \item AF : allele frequency for each ALT allele in the same order as listed: use this when estimated from primary data, not called genotypes
-  \item AN : total number of alleles in called genotypes
-  \item BQ : RMS base quality at this position
-  \item CIGAR : cigar string describing how to align an alternate allele to the reference allele
-  \item DB : dbSNP membership
-  \item DP : combined depth across samples, e.g. DP=154
-  \item END : end position of the variant described in this record (for use with symbolic alleles)
-  \item H2 : membership in hapmap2
-  \item H3 : membership in hapmap3
-  \item MQ : RMS mapping quality, e.g. MQ=52
-  \item MQ0 : Number of MAPQ == 0 reads covering this record
-  \item NS : Number of samples with data
-  \item SB : strand bias at this position
-  \item SOMATIC : indicates that the record is a somatic mutation, for cancer genomics
-  \item VALIDATED : validated by follow-up experiment
-  \item 1000G : membership in 1000 Genomes
-  \item $\ldots$ see Section~\ref{sv-info-keys} for a list of INFO keys reserved for structural variants.
-\end{itemize}
+  INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: $<$key$>$=$<$data$>$[,data]. 
+  INFO keys must match the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed. Arbitrary keys are permitted, although the sub-fields listed in Table~\ref{table:reserved-info} are reserved (albeit optional).
+  
+  \begin{table}[htbp]
+    \centering
+    \begin{tabularx}{\textwidth}{ | l | l | l | X | }
+	Field		& Number	& Type		& Description \\ \hline
+	AA		& 1		& String	& Ancestral allele \\
+	AC		& A		& Integer	& Allele count in genotypes, for each ALT allele, in the same order as listed  \\
+	AD		& R		& Integer	& Total read depth for each allele \\
+	ADF		& R		& Integer	& Read depth for each allele on the forward strand \\
+	ADR		& R		& Integer	& Read depth for each allele on the reverse strand \\
+	AF		& A		& Float		& Allele frequency for each ALT allele in the same order as listed (use this when estimated from primary data, not called genotypes) \\
+	AN		& 1		& Integer	& Total number of alleles in called genotypes \\
+	BQ   		& 1		& Float		& RMS base quality \\
+	CIGAR		& A		& String	& Cigar string describing how to align an alternate allele to the reference allele \\
+	DB		& 0		& Flag		& dbSNP membership \\
```

Comment:
This originally was not number=0, and could have been an A-flag per-allele. The introduction of this table sets it to a particular use-case.